### DISCUSSION TITLE
without joyride__beacon tour open  tour defaulty

### DISCUSSION THREAD

**Discussion by Manoj-1024** [gilbarbara/react-joyride#1006] [Q&A]
can anyone help me out!  joyride__beacon should not be there direct open dialog

**Comment by Manoj-1024** [comment]
Solved

**Reply by eric2788** [reply]
~~may I know how you fix that ?~~

I know the solution, from [here](https://docs.react-joyride.com/migration#removed-breaking)

```
autoStart (use the disableBeacon prop on the first step)
```

**Reply by tayloraucoin** [reply]
@eric2788 

`autoStart` does not appear in the latest docs https://docs.react-joyride.com/props

<img width="662" alt="Screen Shot 2024-06-14 at 11 59 20 AM" src="https://github.com/gilbarbara/react-joyride/assets/10913613/360e56ed-c2d9-4dd4-9c7c-ce922452082f">

**Reply by gilbarbara** [reply]
Hey @tayloraucoin 

If you click the link @eric2788 posted, you'll see that the `autoStart` prop was replaced with `disableBeacon` on the step instead.

**Comment by tayloraucoin** [comment]
@Manoj-1024   how did you solve this?
